documentation/how-to-guides/deploy_magma_access_gateway.md

```diff
@@ -24,15 +24,20 @@ Magma Access Gateway using [snap](https://snapcraft.io/magma-access-gateway).
 - Ubuntu 20.04 LTS
   ([Ubuntu installation guide](https://help.ubuntu.com/lts/installation-guide/amd64/index.html))
 
+> **NOTE**: Unless specified otherwise, all actions in this guide are executed using `root` 
```

Comment:
I'd put this in the install section, not in the system requirements.

Reply:
I've actually just extracted it here, since it applies to the whole guide not only the install part. 
Maybe increased spacing between this note and system requirements wouldn't make it look like part of system requirements?

Reply:
If we move the content of this doc to the `README.md`, then we could have the Usage section and this would be right below, before the Install sub-section

Reply:
Done.